prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Creation</title>
+</head>
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    var $container = document.querySelector('div');
+    console.log($container);
+    document.body.onload = addElement;
+    function addElement() {
+      var $heading = document.createElement('h1');
+      $heading = document.createTextNode('Hello world!');
+      console.log($heading);
```

Comment:
Hi Lisa, for this part we want to put the text into the heading element so that we end up appending the heading element to the page.

Reply:
Gotcha!